ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/precompiles/BigIntegerModularExponentiationPrecompiledContract.java

```diff
@@ -108,7 +85,7 @@ public Bytes compute(final Bytes input, final MessageFrame messageFrame) {
   }
 
   // Equation to estimate the multiplication complexity.
-  private static BigInteger multiplicationComplexity(final BigInteger x) {
+  public static BigInteger multiplicationComplexity(final BigInteger x) {
```

Comment:
We should also consider all methods that were made public to be migrated to the gas calculator.  Some of them are used in the calculation so they should stay.